apps/jubmoji-quest/src/pages/jubmojis/index.tsx

```diff
@@ -122,25 +122,23 @@ export default function JubmojisPage() {
       return <Message>No results found.</Message>;
     }
 
-    if (jubmojis.length !== 0)
```

Comment:
Do we have an issue tracking this fix?

Reply:
Open now a issue for this, and will implement when the design is ready. thk!